### DISCUSSION TITLE
The error "A breakpoint has been reached" is showing on production

### DISCUSSION THREAD

**Discussion by Anoud95** [cefsharp/CefSharp#4549] [General]
### Is there an existing issue for this?

- [X] I have searched both open/closed issues, no issue already exists.

### CefSharp Version

49.0.1

### Operating System

Windows 10

### Architecture

x64

### .Net Version

4.6

### Implementation

WPF

### Reproduction Steps

Randomly 

### Expected behavior

The error message should not be shown 

### Actual behavior

An error message will be shown 
"A breakpoint has been reached"

### Does this problem also occur in the CEF Sample Application

Yes using WPF/OffScreen command line args

**Comment by Anoud95** [comment]
Can you please help, i'm facing a lot of pressure at the customer site

**Comment by Anoud95** [comment]
Can you please give me any hint of the cause of this issue ??

**Comment by amaitland** [comment]
> ### CefSharp Version
> 49.0.1

Please don't open issues for old unsupported versions.

There's  some troubleshooting tips at https://github.com/cefsharp/CefSharp/wiki/Trouble-Shooting

I'd suggest upgrading to a supported version.
